internal/route/routes.go

```diff
@@ -93,40 +95,29 @@ func HandleTodos(w http.ResponseWriter, r *http.Request) {
 
 // HandleTodoDone marks todo as done or undone
 func HandleTodoDone(w http.ResponseWriter, r *http.Request) {
-	if r.Method == "GET" {
+	if r.Method != "PUT" {
 		w.WriteHeader(http.StatusForbidden)
-		log.Fatalf("[error] Route only handles POST requests")
+		log.Fatalf("[error] Route only handles PUT requests")
```

Comment:
Changed to a PUT because we're updating. Not a big deal. I'm an ass.

Reply:
Oh also checking against the one Method we do accept. Instead of one of many that we'd reject. That way GET, DELETE, POST, etc... all get rejected.